azure-pipelines.yml

```diff
@@ -104,6 +104,11 @@ jobs:
       - template: ./.pipelines/artifacts.yml
         parameters:
           Artifacts: $(Artifacts)
+      - task: RichCodeNavIndexer@0
+        inputs:
+          serviceConnection: 'omnisharp-rich-code-nav'
+          githubServiceConnection: 'github'
+          languages: csharp
```

Comment:
just had a thought, can we add `continueOnError: true` just below `task:` but before `inputs:`?

That way this can't break builds if it has an issue internally.

Reply:
Yup, that's a great idea. I'll also update the docs to recommend this